src/include/daos/compression.h

```diff
@@ -26,6 +26,12 @@
 
 #include <daos_prop.h>
 
+#define DC_STATUS_OK			0
```

Comment:
I think it will be easier on the caller of these functions if the error codes are already mapped to daos error codes. For example: 
```
#define DC_STATUS_ERR -DER_MISC
#define DC_STATUS_INVALID_LEVEL -DER_INVAL
#define DC_STATUS_OVERFLOW -DER_TRUNC
#define DC_STATUS_NOMEM -DER_NOSPACE
```
You can find all the currently existing error codes in src/include/daos_errno.h.

Reply:
Fixed, thanks!